testing/ios/IosUnitTests/IosUnitTests.xcodeproj/project.pbxproj

```diff
@@ -487,7 +487,6 @@
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES;
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++17";
-				CLANG_ENABLE_OBJC_ARC = NO;
```

Comment:
The test dylib is built with ARC from the BUILD.gn, so this doesn't do anything.  But I found setting it explicitly to NO confusing, and it's unnecessary.